src/main/webapp/app/course/manage/course-management-tab-bar/course-management-tab-bar.component.ts

```diff
@@ -128,16 +128,31 @@ export class CourseManagementTabBarComponent implements OnInit, OnDestroy {
         this.router.navigate(['/course-management']);
     }
 
+    /**
+     * Checks if the current route contains 'tutorial-groups'.
+     * @return true if the current route is part of the tutorial management
+     */
     shouldHighlightTutorialsLink(): boolean {
         const tutorialsRegex = /tutorial-groups/;
         return tutorialsRegex.test(this.router.url);
     }
 
+    /**
+     * Checks if the current route contains 'grading-system' or 'plagiarism-cases' but not 'exams'.
+     * @return true if the current route is part of the assessment management
+     */
     shouldHighlightAssessmentLink(): boolean {
-        const assessmentLinkRegex = /grading-system|plagiarism-cases/;
-        return assessmentLinkRegex.test(this.router.url);
+        // Exclude exam related links from the assessment link highlighting
+        // example that should not highlight the assessment link: /course-management/{courseId}/exams/{examId}/grading-system/interval
+        const excludeRegex = /exams/;
+        const assessmentLinkRegex = /grading-system|plagiarism-cases|assessment-dashboard/;
+        return assessmentLinkRegex.test(this.router.url) && !excludeRegex.test(this.router.url);
```

Comment:
What about using this pattern? You'd only have to run a single test then

```suggestion
        const assessmentLinkRegex = /^(?!.*exams).*(grading-system|plagiarism-cases|assessment-dashboard)/;
        return assessmentLinkRegex.test(this.router.url);
```

Reply:
I tested it locally and it works. Thank you for your suggestion :)